client/state/sites/action-types.js

```diff
@@ -0,0 +1 @@
+export const RECEIVE_SITE = 'RECEIVE_SITE';
```

Comment:
In other parts of the codebase we are using `keymirror`, should we do the same here?